### PR TITLE
Remove <table> from config screen for 2.266

### PR DESCRIPTION
**Pull request by REBELinBLUE**
I've never done anything with the Jenkins code before but I thought I'd help it along, let me know if there is anything else I should do

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/environmentdashboard/DashboardBuilder/config.jelly [hunk 1]
```diff
@@ -1,44 +1,38 @@
...
-                </f:optionalBlock>
-            </table>
-        </f:nested>
+	<f:entry title="Package Name" field="packageName">
+		<f:textbox />
+	</f:entry>
 
+	<f:nested>
+		<f:optionalBlock field="addColumns" title="Add Custom Columns" inline="true">
+			<f:entry>
```
Comment (timja) [inline src/main/resources/org/jenkinsci/plugins/environmentdashboard/DashboardBuilder/config.jelly:23]: there's migration guides here: https://www.jenkins.io/doc/developer/views/table-to-div-migration/

basically:

table => div
tr => div class="tr"
td => div

and keep the rest of the markup the same (not strictly needed if you're dropping support for old core, but normally easier to stay compatible if you don't change the markup too)

you'll want to use a local taglib so that the right tags will be used where required

here's a full example doing it for table:
https://github.com/jenkinsci/git-plugin/pull/973/files

duplicate the tag pretty much for tr and td, we normally call it rowWrapper and entryWrapper

also you're IDE has changed all  the tabs to spaces would be good if that could be avoided

here's another example where they are extracted into a standalone taglib:
https://github.com/jenkinsci/postbuild-task-plugin/pull/6

Comment (REBELinBLUE) [reply src/main/resources/org/jenkinsci/plugins/environmentdashboard/DashboardBuilder/config.jelly:23]: Ah cool I will take a look, I was just experimenting with getting our work install running properly, I will make these changes in the morning

**Review by timja — APPROVED** [review]
this would work assuming the maintainer is happy to migrate to the latest core

**Comment by REBELinBLUE** [comment]
I see what you mean, I guess if they aren't I will do the `<j:when test="${divBasedFormLayout}">` approach

File: src/main/resources/lib/environmentdashboard/blockWrapper.jelly [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
```
Comment (timja) [inline src/main/resources/lib/environmentdashboard/blockWrapper.jelly:1]: You also need an empty marker file in this directory called ‘taglib’

Comment (REBELinBLUE) [reply src/main/resources/lib/environmentdashboard/blockWrapper.jelly:1]: ah!

**Review by timja — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>environment-dashboard</artifactId>
-  <version>1.1.9-SNAPSHOT</version>
+  <version>1.1.10-SNAPSHOT</version>
```
Comment (res0nance) [inline pom.xml:13]: Revert this change as the next version is in fact 1.1.9 occording to tags

**Review by res0nance — APPROVED** [review]
Changes look good

File: pom.xml (hunk 3 above)
Comment (REBELinBLUE) [reply pom.xml]: Done

**Review by res0nance — APPROVED** [review]

**Comment by timja** [comment]
@willwh

**Review by oleg-nenashev — APPROVED** [review]

**Comment by jimklimov** [comment]
Confirming that with a plugin built from merger of this PR and current master branch (and TBH a couple other rebuilt suspects), my UI problems with legacy job configs in 2.269 went away.

**Comment by willwh** [comment]
Hi folks, Thanks much for this. Apologies for the late response! I don't have an environment set up to publish changes, I'd love to get this merged and released though. I'll try to find the time this weekend!

**Comment by jimklimov** [comment]
Cool! Just in case, the doc: https://www.jenkins.io/doc/developer/publishing/releasing/
